src/qt/bitcoingui.cpp

```diff
@@ -392,6 +393,9 @@ void BitcoinGUI::createActions()
     openAction = new QAction(platformStyle->TextColorIcon(":/icons/open"), tr("Open &URI..."), this);
     openAction->setStatusTip(tr("Open a dogecoin: URI or payment request"));
 
+    importPrivateKeyAction = new QAction(platformStyle->TextColorIcon(":/icons/address-book"), tr("&Import Private Key..."), this);
```

Comment:
Non-blocking: Would love to have a new icon for this. Can merge regardless now though, we can change it later too.